### ISSUE TITLE
OUI Logo should change appearance depending on light/dark mode

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#905]

**Comment by joshuarrrr** [comment]
OpenSearch logo? I think we'll need screenshots here to make sure we account for it everywhere it matters.

**Comment by KrooshalUX** [comment]
@joshuarrrr if we can control it through utilizing the logoOpenSearch if possible, I think that would resolve many issues. I am not sure if this is going to require a campaign across the feature plug-ins or not, though, since in some instances they may be using additional OpenSearch logo assets.

**Comment by joshuarrrr** [comment]
Oh, so this issue is just for OUI doc site? Yeah, shouldn't be a problem.

**Comment by KrooshalUX** [comment]
No, sorry I see how the wording might be confusing. The change is to make logoOpenSearch as an icon change depending on light or dark mode. That way when anyone in OSD uses logoOpenSearch , the change is automatic. 

Ideally we would campaign all non OUI driven uses of logoOpenSearch to be OUI driven. (And also adding the full logo as well). This is how EUI served up logos in the past for Kbn.

The changes also need to update in the doc site as well (ex: the logo in the nav bar)

**Comment by KrooshalUX** [comment]
That makes sense to me - is there much risk to the logo being misused/ overrides giving the logo color incorrectly if we are providing it as monocrhome and compatible with the color prop?

**Comment by AMoo-Miki** [comment]
Coloring the monochrome ones would only be able to set the entire logo into a specific color; I am having a hard time making up an abusive single color; misuse per the usage guidance for contrasting background would be possible but I don't believe OUI should concern itself with that much detail.

**Comment by anirudha** [comment]
Closing — superseded by the V9 theme initiative (#1695) and OUI 2.0 work. The new theme system addresses the goals of this issue. See the V9 RFC for the current direction.
